functions/go/set-labels/README.md

```diff
@@ -38,47 +54,45 @@ data:
   fruit: apple
 ```
 
-You can use key `fieldSpecs` to specify the resource selector you want to use.
-By default, the function will not only add or update the labels in
-`metadata/labels` but also a bunch of different places where have references to
-the labels. These field specs are defined in
-https://github.com/kubernetes-sigs/kustomize/blob/master/api/konfig/builtinpluginconsts/commonlabels.go#L6
-
-You need to use a custom resource to specify additional information.
+To use a `SetLabels` custom resource as the `functionConfig`, the desired labels
+must be specified in the `labels` field. You can optionally
```

Comment:
For introducing `additionLabelFields`, we can say something along the lines of:

Sometimes you have resources (especially custom resources) that have labels fields in fields other than the defaults (link-to-common-labels), you can specify such label fields using `additionalLabelFields`.